src/scss/_footer.scss

```diff
@@ -41,19 +55,35 @@
       padding: 50px 65px 45px;
 
       background-image: url('../images/background1.png');
+
+      @media screen and (max-width: $xl) {
```

Comment:
this is for `&__tariff`, please move it after line 52

Reply:
![q1](https://user-images.githubusercontent.com/62384781/184701555-ef9bbdc7-4722-4bc8-82b8-c33549616470.png)